### PR TITLE
Auto increment (Serial) ID attribute added

### PR DESCRIPTION
**Pull request by rmeena840**
Auto increment (Serial) ID attribute added
Task: https://www.pivotaltracker.com/story/show/182306016

### CODE REVIEW

**Comment by rmeena840** [comment]
@robert-zaremba Please review the PR.

File: examples/dao/setup_sigma_dao.sql [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 -- create a sigma_daos table if does not exit already.
 CREATE TABLE IF NOT EXISTS sigma_daos (
-  app_id bigint PRIMARY KEY, -- application id
+  id SERIAL PRIMARY KEY, -- auto increment id
+  app_id bigint, -- application id
```
Comment (robert-zaremba) [inline examples/dao/setup_sigma_dao.sql:8]: we need to have an index by `app_id` as well.

**Review by robert-zaremba — APPROVED** [review]
pre approving. Please add a secondary index for `app_id`

File: examples/dao/setup_sigma_dao.sql (hunk 1 above)
Comment (rmeena840) [reply examples/dao/setup_sigma_dao.sql:8]: Pinged you. Please check.

File: examples/dao/setup_sigma_dao.sql [hunk 2]
```diff
@@ -4,7 +4,8 @@
...
   app_params jsonb, -- application params with approval etc.
   asset_id bigint -- token id
 );
```
Comment (robert-zaremba) [inline examples/dao/setup_sigma_dao.sql:11]:
```suggestion
);
CREATE UNIQUE INDEX sigma_daos_app_id_idx ON sigma_daos (app_id);
```

Comment (robert-zaremba) [reply examples/dao/setup_sigma_dao.sql:11]: this index should work, please test it.

File: examples/dao/setup_sigma_dao.sql (hunk 1 above)
Comment (robert-zaremba) [reply examples/dao/setup_sigma_dao.sql:8]: provided example in a comment below.

Comment (rmeena840) [reply examples/dao/setup_sigma_dao.sql:8]: Tested.

**Pull request merged**